operator/v1/types_ingress.go

```diff
@@ -121,6 +121,14 @@ type IngressControllerSpec struct {
 	//
 	// +optional
 	NodePlacement *NodePlacement `json:"nodePlacement,omitempty"`
+
+	// SecuritySpec is used to specify the settings for securing IngressController
```

Comment:
Should be capitalized as `securitySpec`.  Note that all godoc should be referring to fields using the json capitalization (i.e., with a small first letter).

Could simplify: "is used to specify"→"specifies".

Reply:
I'll update. Thanks for the pointer. I was confused on when to capitalize the first letter of a field because I see a mixture in the api. For example, the doc for `EndpointPublishingStrategy` is `EndpointPublishingStrategy` instead of `endpointPublishingStrategy`:
```
// EndpointPublishingStrategy is a way to publish the endpoints of an
// IngressController, and represents the type and any additional configuration
// for a specific type.
type EndpointPublishingStrategy struct {
<SNIP>
}
```
Is lowercase capitalization only required for fields of `spec` and `status`?

Reply:
References to the type `EndpointPublishingStrategy` should use an initial upper-case letter.  References to fields inside `EndpointPublishingStrategy` (i.e., `type` or `loadBalancer`) should use lower-case letters.

Reply:
Thanks for the explanation. I have updated the docs based on your feedback.